### PR TITLE
Update nitdmreader.yml

### PR DESCRIPTION
**Pull request by jjaco2**
Made a few low-level changes (tense, punctuation, rewrites for minimalism/clarity).

- [ ] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/systemlink-OpenAPI-documents/blob/master/CONTRIBUTING.md).

TODO: Check the above box with an 'x' indicating you've read and followed [CONTRIBUTING.md](https://github.com/ni/systemlink-OpenAPI-documents/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Improvement of descriptions and summaries.

### Why should this Pull Request be merged?

Consistency among API docs.

### What testing has been done?

NA

### CODE REVIEW

File: tdm-reader/nitdmreader.yml [hunk 1]
```diff
@@ -506,13 +506,13 @@ paths:
     post:
       tags: [data]
       summary: Returns data from channels.
-      description: Queries files based on channel and window specifications and returns channel data that matches the query.  The channel specifications select channels by group and channel relationships.  Window specifications determine a bounding box of minimum and maximum x and y values, as well as specifying the type of decimation performed on data within the window.
+      description: Queries files based on channel and window specifications and returns channel data that matches the query. The channel specifications select channels by group and channel relationships. Window specifications determine a bounding box of minimum and maximum x and y values, as well as specifying the type of decimation performed on data within the window.
```
Comment (maxgax) [inline tdm-reader/nitdmreader.yml]:
```suggestion
      description: Queries files based on channel and window specifications and returns channel data that matches the query. The channel specifications select channels by group and channel relationships. Window specifications determine a bounding box of minimum and maximum x and y values, as well as specify the type of decimation performed on data within the window.
```

File: tdm-reader/nitdmreader.yml [hunk 2]
```diff
@@ -196,17 +196,17 @@ definitions:
...
         type: array
         items:
-          description: Y channel(s) and an optional x channel
+          description: y channel(s) and an optional x channel
           properties:
             xChannel:
               $ref: '#/definitions/OneChannelSpecifier'
             yChannels:
-              description: 'Channels to be treated as y channel, associated with x channel'
+              description: 'Channels to be treated as y channels associated with x channel'
```
Comment (maxgax) [inline tdm-reader/nitdmreader.yml:209]:
```suggestion
              description: Channels to be treated as y channels associated with 'xChannel'
```

File: tdm-reader/nitdmreader.yml [hunk 3]
```diff
@@ -112,13 +112,13 @@ definitions:
...
       properties:
-        description: Property values.
+        description: Property values
         type: array
         items:
           $ref: '#/definitions/Property'
       totalCount:
         type: integer
-        description: Count of all properties. This could be more than the items returned in Properties.
+        description: Count of all properties. This might be more than the items returned in Properties.
```
Comment (maxgax) [inline tdm-reader/nitdmreader.yml:121]: I kind of like 'may' here. What do you think? If you agree, there is an instance of 'might' on line 291 as well.

File: tdm-reader/nitdmreader.yml [hunk 4]
```diff
@@ -134,39 +134,39 @@ definitions:
     properties:
       decimatex:
         type: integer
-        description: Number of values on the x-axis. This determines the number of points that can be displayed.
+        description: Number of values on the x axis. This determines the number of points that can be displayed.
```
Comment (maxgax) [inline tdm-reader/nitdmreader.yml:137]: Are we removing the '-' for consistency w/ other docs? I thought x-axis was more correct.

File: tdm-reader/nitdmreader.yml [hunk 5]
```diff
@@ -175,7 +175,7 @@ definitions:
       showPoints:
         type: boolean
         default: false
-        description: Represent data as points. When decimation is performed, the data returned represents all of the points in that x-axis increment.
+        description: Represent data as points. When decimation is performed, the data returned represents all the points in that x-axis increment.
```
Comment (maxgax) [inline tdm-reader/nitdmreader.yml:178]: Missed one.
```suggestion
        description: Represent data as points. When decimation is performed, the data returned represents all the points in that x axis increment.
```

File: tdm-reader/nitdmreader.yml (hunk 3 above)
Comment (jjaco2) [reply tdm-reader/nitdmreader.yml:121]: "May" is fine too, in this case.

Also, I'm so sorry for the delay. I updated my notification settings so I won't miss things like this moving forward.

**Review by jjaco2 — COMMENTED** [review]
Responding to one question and resolving some conversations (is that okay for me to do at this stage?) So sorry I didn't see this before now!

**Review by maxgax — APPROVED** [review]

**Pull request merged**